pages/linux/hcitool.md

```diff
@@ -0,0 +1,36 @@
+# hcitool
+
+> Monitor, configure connections, and send special commands to Bluetooth devices.
+> More information: <https://manned.org/hcitool>.
+
+- Scan for Bluetooth devices:
+
+`hcitool scan`
+
+- Output the name of a device specifying its MAC address:
```

Comment:
```suggestion
- Output the name of a device, returning its MAC address:
```

Reply:
Wait does this give the mac address, or want the mac address?

Reply:
It give the mac address.